standalone/src/models/product-selection/product-selection-draft/presets/sample-data-b2b/default-product-selection.ts

```diff
@@ -1,9 +1,39 @@
 import { LocalizedStringDraft } from '../../../../commons';
-import type { TProductSelectionDraftBuilder } from '../../../types';
-import * as ProductSelectionDraft from '../../index';
+import type { TBuilder } from '../../../../../core';
+import { TCtpProductSelectionMode } from '../../../../../graphql-types';
```

Comment:
Should we now use relative paths instead of `@commercetools-test-data/graphql-types` here and for all the imports alike?

Reply:
At this moment we need to use relative paths but we will move to aliases once [this PR](https://github.com/commercetools/test-data/pull/814) is merged.